base/dcp_sharded.go

```
@@ -352,7 +352,10 @@ func startHeartbeater(bucket Bucket, cbgtContext *CbgtContext) (Heartbeater, err
 	// TODO: Allow customization of heartbeat interval
 	intervalSeconds := 1
 	Debugf(KeyDCP, "Sending CBGT node heartbeats at interval: %vs", intervalSeconds)
-	heartbeater.StartSendingHeartbeats(intervalSeconds)
+	err = heartbeater.StartSendingHeartbeats(intervalSeconds)
+	if err != nil {
+		Debugf(KeyDCP, "Error start sending heartbeats: %v", err)
```

In this case it feels like we should return the error up

---

Fixed in latest revision.